test/test_cmd.py

```diff
@@ -42,6 +42,7 @@ def call_command(self, argv=None):
         Arguments:
             argv (list) -- arguments to pass to the edx_lint script
         """
+        self    # pylint: disable=pointless-statement
```

Comment:
Same as the comment above, if this is not useless, A comment would help and if it is useless, then removing the line would be good.